### PR TITLE
add aggevm wallet

### PR DESCRIPTION
**Pull request by zhangzhishun**

### CODE REVIEW

**Comment by zhangzhishun** [comment]
Sry. I don't know what should I need to do.

**Comment by HeesungB** [comment]
@zhangzhishun Hello
It would be helpful to read this [link](https://github.com/chainapsis/keplr-chain-registry#requirement-details).

and You can test on local using below command.
```
yarn install
yarn validate cosmos/aggevm.json
```

File: cosmos/aggevm.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "website":"https://web3-idea.xyz"
+  },
+  "chainId": "aggevm",
+  "chainName": "Agg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "AGG",
+    "coinMinimalDenom": "uagg",
+    "coinDecimals": 6,
+    "coinGeckoId": "agg",
```
Comment (HeesungB) [inline cosmos/aggevm.json:16]: @zhangzhishun Could you remove `coinGeckoId` field? because It is invalid

File: cosmos/aggevm.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/agg",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/agg",
```
Comment (HeesungB) [inline cosmos/aggevm.json:20]: @zhangzhishun Could you remove `walletUrl` and `walletUrlForStaking` field?

File: cosmos/aggevm.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "bech32PrefixValPub": "aggvaloperpub",
+    "bech32PrefixConsAddr": "aggvalcons",
+    "bech32PrefixConsPub": "aggvalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "AGG",
+      "coinMinimalDenom": "uagg",
+      "coinDecimals": 6,
+      "coinGeckoId": "agg",
```
Comment (HeesungB) [inline cosmos/aggevm.json:37]: @zhangzhishun Could you remove coinGeckoId field? because It is invalid

File: cosmos/aggevm.json (hunk 1 above)
Comment (zhangzhishun) [reply cosmos/aggevm.json:16]: Thanks.

But I received the message: "stakeCurrency.coinDenom" is required and "stakeCurrency.coinDenom" is not allowed to be empty after removing it.

Comment (HeesungB) [reply cosmos/aggevm.json:16]: @zhangzhishun It is ok that you remove `coinGeckoId`field!

File: cosmos/aggevm.json (hunk 3 above)
Comment (zhangzhishun) [reply cosmos/aggevm.json]: I do it. Thanks

File: cosmos/aggevm.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  "nodeProvider": {
+    "name": "Web3 Idea",
+    "email": "zhangzhishun1024@gmail.com",
+    "website":"https://web3-idea.xyz"
+  },
+  "chainId": "aggevm",
+  "chainName": "Agg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
+  "stakeCurrency": {
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
```
Comment (HeesungB) [inline cosmos/aggevm.json:13]: @zhangzhishun Could you add `stakeCurrency` information?
and `coinImageUrl` should be `uagg.png` better

File: cosmos/aggevm.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    "bech32PrefixAccAddr": "agg",
+    "bech32PrefixAccPub": "aggpub",
+    "bech32PrefixValAddr": "aggvaloper",
+    "bech32PrefixValPub": "aggvaloperpub",
+    "bech32PrefixConsAddr": "aggvalcons",
+    "bech32PrefixConsPub": "aggvalconspub"
+  },
+  "currencies": [
+    {
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
```
Comment (HeesungB) [inline cosmos/aggevm.json:28]: @zhangzhishun Could you add `currencies ` information?
and `coinImageUrl` should be `uagg.png` better

File: cosmos/aggevm.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "AGG",
+      "coinMinimalDenom": "uagg",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
```
Comment (HeesungB) [inline cosmos/aggevm.json:36]: @zhangzhishun `coinImageUrl` should be uagg.png better

**Pull request merged**